src/scenes/AcceleratorRouter/ParticipantProjects/Reports/ReportView.tsx

```diff
@@ -0,0 +1,199 @@
+  const [selectedReport, setSelectedReport] = useState<AcceleratorReport>();
+  const { isAllowed } = useUser();
+  const [showApproveDialog, setShowApproveDialog] = useState<boolean>(false);
+  const [showRejectDialog, setShowRejectDialog] = useState<boolean>(false);
+  const { participantProject } = useParticipantProjectData();
+  const theme = useTheme();
+  const [editingId, setEditingId] = useState<string | undefined>();
+
+  useEffect(() => {
+    if (projectId) {
+      const request = dispatch(requestListAcceleratorReports({ projectId, includeFuture: true, includeMetrics: true }));
+      setRequestId(request.requestId);
+    }
+  }, [projectId]);
+
+  useEffect(() => {
+    if (reportsResults?.status === 'error') {
+      return;
+    }
+    if (reportsResults?.data) {
+      setReports(reportsResults.data);
+    }
+  }, [reportsResults]);
+
+  useEffect(() => {
+    if (reports) {
+      const reportSelected = reports.find((report) => report.id.toString() === reportId);
+      setSelectedReport(reportSelected);
+    }
+  }, [reportId, reports]);
+
+  const crumbs: Crumb[] = useMemo(
+    () => [
+      {
+        name: activeLocale ? strings.REPORTS : '',
+        to: APP_PATHS.ACCELERATOR_PROJECT_REPORTS,
+      },
+    ],
+    [activeLocale]
+  );
+
+  const callToAction = useMemo(() => {
+    return (
+      isAllowed('UPDATE_SUBMISSION_STATUS') && (
+        <>
+          <Button
+            disabled={selectedReport?.status === 'Needs Update'}
+            id='rejectDeliverable'
+            label={strings.REJECT_ACTION}
+            priority='secondary'
+            onClick={() => void setShowRejectDialog(true)}
+            size='medium'
+            type='destructive'
+          />
+          <Button
+            disabled={selectedReport?.status === 'Approved'}
+            id='approveDeliverable'
+            label={strings.APPROVE}
+            onClick={() => void setShowApproveDialog(true)}
+            size='medium'
+          />
+        </>
+      )
+    );
+  }, [selectedReport?.status, isAllowed]);
+
+  const rightComponent = useMemo(
+    () => (
+      <Box display='flex' flexDirection='row' flexGrow={0} marginRight={theme.spacing(3)} justifyContent='right'>
+        {callToAction}
+      </Box>
+    ),
+    [callToAction]
+  );
+
+  const year = selectedReport?.startDate.split('-')[0];
+  const quarterNumber = selectedReport?.startDate
+    ? Math.ceil((new Date(selectedReport?.startDate).getMonth() + 1) / 3)
+    : 0;
+  const reportName =
+    selectedReport?.frequency === 'Annual' ? `${year}` : quarterNumber ? `${year}-Q${quarterNumber}` : '';
+
+  return (
+    <Page
+      title={
+        <TitleBar
+          title={`${strings.REPORT} (${reportName})`}
+          subtitle={participantProject ? `${strings.PROJECT}: ${participantProject?.dealName}` : ''}
+        />
+      }
+      rightComponent={rightComponent}
+      crumbs={crumbs}
+    >
+      {/* {requestStatus === 'pending' && <BusySpinner />} */}
```

Comment:
Update or remove?

Reply:
Will remove it and add it in redux PR if necessary